scripts/dts/python-devicetree/src/devicetree/edtlib.py

```diff
@@ -2053,6 +2053,49 @@ def scc_order(self) -> List[List[Node]]:
         except Exception as e:
             raise EDTError(e)
 
+    def _link_properties(self, parent_node, node):
```

Comment:
these changes likely deserve some extra test cases?

Reply:
Yes, sorry forgot that there's a test suite for this - added a new test case for this one in the first patch.